lua/starfall/libs_cl/file.lua

```diff
@@ -43,13 +44,23 @@ do
 		for k, plyid in ipairs(dirs) do
 			local dir = "sf_filedatatemp/"..plyid
 			files = file.Find(dir.."/*", "DATA")
-			if next(files)==nil then SF.DeleteFolder(dir) else
-				for k, filen in ipairs(files) do
-					local path = dir.."/"..filen
-					local time, size = file.Time(path, "DATA"), file.Size(path, "DATA")
+			local fileCount = #files
+			for k, filen in ipairs(files) do
+				local path = dir.."/"..filen
+				local time = file.Time(path, "DATA")
+
+				if os.time() > time + cv_temp_maxage:GetInt() then
```

Comment:
Sorry to nitpick a little more, but could you localize `os.time()` and `cv_temp_maxage:GetInt()` before the loops?